tests/system_tests/external_interaction/test_ispyb_dev_connection.py

```diff
@@ -59,16 +61,16 @@ def test_ispyb_deposition_comment_correct_for_3D_on_failure(
     dummy_ispyb_3d: Store3DGridscanInIspyb, fetch_comment
 ):
     dcid = dummy_ispyb_3d.begin_deposition()
-    dcid1 = dcid[0][0]
-    dcid2 = dcid[0][1]
+    dcid1 = dcid.data_collection_ids[0]  # type: ignore
+    dcid2 = dcid.data_collection_ids[0]  # type: ignore
```

Comment:
Should: Do we have no way of checking the second deposition now?

Reply:
oops...